### ISSUE TITLE
Feature request: Sort order for discovered objects

### ISSUE THREAD

**Issue by Comradin** [joncage/ed-scout#186]
When scanning a system, ed-scout sorts these entries after their appearance  in the log file.

Only the real valuable planets like earth like and the water worlds are put on top of the list. I guess if their value is above 1 million credits? But most systems we scan do not have these types of bodies in them. So I tend to do a detailed surface scan on objects that have at least an estimation of >200k credits. Most times these are the metal content worlds.

The Problem is, if the system contains a large number of belts and low value planets, its hard to scan for these objects in the list. Even worse if you are playing on full screen mode when switching widows to the second screen.

I suggest to sort the ordering of the planets after their estimated scan value as soon as the FSSAllBodiesFound event gets logged. This way I can easily decide which planets to visit for a detailed surface scan to make some extra money from a system that's less interesting but crowded with dozens of objects.

**Comment by joncage** [comment]
Yep, sorted after all are scanned sounds like a sensible addition. Could potentially make it a config option as to what it does and/or make the 'valuable body' value configurable (from memory I think it's currently set to 500k).

**Comment by Comradin** [comment]
Hmm,

I had some systems with objects ranging from 450k to 200k that got my interest for a detailed surface scan. But yeah, that would be awesome.

In terms of awesomeness, your software is awesome!! Its a very good companion app when doing exploration. Many thanks!! And enjoy your coffee :heart_decoration:

**Comment by davidknapik** [comment]
Bump for this request. In addition to a value/sort order. I think it would also be nice to be able to visually see if the body has been scanned (change the highlight/add a column etc...) 

Adding a configurable value in the ini for scan value would be great too.
